core/templates/dev/head/components/sharing_links_directive.html

```diff
@@ -1,40 +1,76 @@
 <script type="text/ng-template" id="components/sharingLinks">
   <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
     <li>
-      <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" ng-click="registerShareExplorationEvent('gplus')" target="_window">
+      <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes');" ng-click="registerShareExplorationEvent('gplus');" target="_window">
```

Comment:
So in the dashboard, they actually weren't working. Clicking on the embed button, for example, showed the modal but then the parent window went into the editor. I'll change these backs but it may be something to stay aware of.